src/components/CommentList.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { useEffect, useState } from 'react';
+import * as api from '../api';
+
+export default function CommentList({ review_id }) {
+  const [comments, setComments] = useState([]);
+  useEffect(() => {
+    api.fetchCommentsByReviewId(review_id).then((comments) => {
+      setComments(comments);
+    });
+  }, [review_id]);
+
+  if (!comments.length)
+    return (
+      <div>
+        <h2>Comments for review {review_id} </h2>
```

Comment:
This is super picky but I don't feel like the user needs to see the review ID, if we're cloning reddit essentially, each post would have an ID in the database but it's not shown to the user

Reply:
Ah yeah, that was for my benefit making sure that it was working correctly - will remove it now!